sensa/client.py

```diff
@@ -626,27 +626,27 @@ def from_message(msg, verify_ssl_cert=None) -> Client:
 
     @staticmethod
     def local(basedir=None):
-        """Create a Local Cortex implementation (mock)
+        """Create a Local Sensa implementation (mock)
 
         :param basedir: Root filesystem location, defaults to None
         :type basedir: str, optional
-        :return: an instance of :class:`cortex.client.Local`
-        :rtype: :class:`cortex.client.Local`
+        :return: an instance of :class:`sensa.client.Local`
+        :rtype: :class:`sensa.client.Local`
         """
         return Local(basedir)
 
     @staticmethod
     def login():
         """
-        Login to Cortex6. The function prompts the caller for Cortex Personal Access Config.
+        Login to Sensa6. The function prompts the caller for Sensa Personal Access Config.
 
         **Example**
 
-        >>> Cortex.login()
-        Cortex Personal Access Config: Cortex Personal Access Config
-        Cortex Project: The project that you to start using you Cortex assets from. (Not required)
+        >>> Sensa.login()
+        Sensa Personal Access Config: Sensa Personal Access Config
+        Sensa Project: The project that you to start using you Sensa assets from. (Not required)
         """
-        config = input("Cortex Personal Access Config: ")
+        config = input("Sensa Personal Access Config: ")
         project = input("Project: ")
         os.environ["CORTEX_PERSONAL_ACCESS_CONFIG"] = config
```

Comment:
Are any [environment variables](https://cognitivescale.github.io/cortex-fabric/docs/getting-started/authenticate-to-libs#environmental-variables) going to be changing as well to start with `SENSA_` ? There are scattered uses of these variables in the package.

Reply:
Not right now but eventually! This can be discussed but I think the iteration 1 is for the library name, its usage and distribution.